app/lib/hyrax/hirmeos/client.rb

```diff
@@ -16,16 +16,24 @@ def post_work(work)
     id_translation_connection.post('/works', work.to_json)
   end
 
-  def get_work(uuid)
-    id_translation_connection.get("/translate?uri=urn:uuid:#{uuid}")
+  def get_work(hyku_uuid)
+    id_translation_connection.get("/translate?uri=urn:uuid:#{hyku_uuid}")
+  end
+
+  def get_work_identifiers(hirmeos_uuid)
+    id_translation_connection.get("/works?uuid=#{hirmeos_uuid}")
+  end
+
+  def delete_work(hirmeos_uuid)
+    id_translation_connection.delete('/works', uuid: "urn:uuid:#{hirmeos_uuid}")
   end
```

Comment:
Does this work?  I thought Bertie was saying this isn't implemented on the HIRMEOS side.

Reply:
Returns a 500 error, but it is supposed to work:
https://github.com/hirmeos/identifier_translation_service#delete-works-data